container/builder/osbuild-koji.conf

```diff
@@ -1,5 +1,7 @@
 [composer]
-url = http://composer:8701/
+url = https://composer:8701/
+ssl_cert = /share/worker-crt.pem, /share/worker-key.pem
```

Comment:
Oh... this answers my previous comment... it all just gets concatonated, so does not matter what's the key and what's the cert? Feels odd, but if it really does not matter, I guess making an artificial distinction would be even weirder...

Reply:
The distinction wouldn't be that arbitrary. Separating keys from certificates seems to be common practice (for example in in dnf and curl). I agree it's awkward, because they're all `.pem` files which seem to be accepted everywhere.

If we decide to change this, we can in a follow-up without breaking anything.